### DISCUSSION TITLE
Analytics: SelectItem inconsistency in types

### DISCUSSION THREAD

**Discussion by jeffreydelooff** [invertase/react-native-firebase#6500] [Q&A] [answered]
In this [PR](https://github.com/invertase/react-native-firebase/pull/5253/files) the description was used to remove the `content_type` but wasn't actually removed from the types itself. The [official docs](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#select_item) does not have a `content_type` field either.

I have tested sending an event with and without the `content_type` field and luckily Firebase didn't give any errors. From previous testing with standard events I have had errors in the past when using "unsupported"/custom field names.

Is it worth it to make a PR for [this](https://github.com/jeffreydelooff/react-native-firebase/commit/b8b055462668eced65ed0e3be710f99300b08af8) to avoid confusion?

<img width="896" alt="Screenshot 2022-08-31 at 13 46 30" src="https://user-images.githubusercontent.com/263346/187671932-5dba0c80-75ed-42d2-acac-6eb4dcf06b02.png">

**Comment by mikehardy** [comment, marked as answer]
Hey @jeffreydelooff :wave: - that looks spot on as a change + potential PR, would be happy to merge it - I fear since it is removing a type it's a breaking change for many, so it might queue for a little bit waiting for a v16. Perhaps 2 PRs, one with a `@deprecation` the second with the removal ?
